core/account.py

```diff
@@ -51,11 +52,49 @@ def get_account(
     try:
         result = db.session.execute(stmnt)
         account = result.scalars().one()
-        account_schema = AccountSchema()
+        account_schema = AccountSchema()    
         return account_schema.dump(account), 200
     except sqlalchemy.exc.NoResultFound:
         return {"error": "No matching account found"}, 404
 
+def get_bulk_accounts(
+    account_id: list,
+) -> Dict:
+    """
+    Get multiple accounts corresponding to the given account ids
+    and return account object schema and status code
+    :param account_id: (str) the id to search
```

Comment:
if account_id is a list then we can update this docstring param

Reply:
Has been amended to describe accurately